### DISCUSSION TITLE
Will the Fragment objects be merged?

### DISCUSSION THREAD

**Discussion by ChengxiangQiu** [stuart-lab/signac#454] [Q&A] [answered]
I created the Signac object on each individual sample, and then merged them to create a combined object. I noticed that the fragment objects from each individual sample were not merged, but rather saved as a list object. 

...
[[4]]
A Fragment object for 7270 cells
[[5]]
A Fragment object for 13844 cells
[[6]]
A Fragment object for 1610 cells
[[7]]
A Fragment object for 1232 cells
...

My question is: 
1) is it expected (unmerged fragments)? 
2) will it affect any following analysis using the fragment files? 
3) do I need to manually create a merged fragments and then added it to the combined object?

Thank you!

**Comment by timoast** [comment, marked as answer]
In short:

1. Yes
2. No
3. No

Since Signac 1.0.0, we have re-designed how fragment files are handled to better support multiple fragment files and to avoid needing to merge or alter files on-disk. Fragment information is now stored as a list of files, and all functions in Signac that operate on a fragment files will collect information across all files in the list.

**Reply by ChengxiangQiu** [reply]
Thank you! that's a perfect answer!!
